internal/controller/controllers/bmo_utils.go

```diff
@@ -16,7 +16,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const MinimalVersionForConvergedFlow = "4.11.0-0.alpha"
+const MinimalVersionForConvergedFlow = "4.12.0-ec.4"
```

Comment:
Curious if we think this will cause a problem.

Should we leave 4.11 opt-in or just push everyone to use 4.12 for this feature?
If we still want 4.11 maybe we'll need to keep the env var for that?

Reply:
As discussed f2f, lets disable it for 4.11 and just push everyone to use 4.12 for this feature (+ keeping the env var so it's possible to disable it)

Reply:
Yup, that's what the PR does now.